.gitignore

```diff
@@ -1,16 +1,59 @@
-.DS_Store
-www/downloads
-www/uploads
-node_modules
-www/nltk_data
-main_config.json
-graphql_config.json
-map.json
-*.log
+# backup files
+*~
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+# vim swap files
 *.swp
-.idea
 
-# secrets and values files
-*smile.secret.yaml
-*values-clowder.yaml
-*values-smile.yaml
\ No newline at end of file
+# IDEA:
+.idea/
```

Comment:
glad you catch the .idea in the gitingore. There is currently still idea in the root. you might want to also remove that folder 
<img width="285" alt="image" src="https://github.com/ncsa/smm-deployment/assets/13950475/51efe28b-84d9-4918-a07c-285252ef67b7">

Reply:
removed the folder